packages/react-vanilla-components/src/components/common/FieldWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback, useState } from "react";
+import LabelContainer from "./LabelContainer";
+import Description from "./Description";
+import HelpContainer from "./HelpContainer";
```

Comment:
Exporting modules that are used exclusively within a single file can add unnecessary complexity and may make the code less maintainable. We should keep these functional components self contained